converter/src/main/java/gov/cms/qpp/conversion/validate/CpcClinicalDocumentValidator.java

```diff
@@ -23,7 +24,10 @@
 public class CpcClinicalDocumentValidator extends NodeValidator {
 
 	static final String END_DATE_VARIABLE = "CPC_END_DATE";
+	static final DateTimeFormatter END_DATE_FORMAT = DateTimeFormatter.ofPattern("MMMM dd, yyyy");
```

Comment:
What does MMMM do over MM?

Reply:
I believe it expands to a full month name.  MM = 01, MMMM = January.  The formatting rules are rather cryptic: https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html